### ISSUE TITLE
JOSS review issues

### ISSUE THREAD

**Issue by srmnitc** [wasserman-group/pyCADMium_examples#3]
This is part of the review for JOSS at https://github.com/openjournals/joss-reviews/issues/4459 .

The main source of examples for the package is this repository. The repository has a large number of really useful examples. However, there are also some issues which I have listed together below:

- There is no explanation for the inexperienced user on how to use these examples. There are dependencies which are not listed such as `matplotlib`, `plotly` and `ipyml`
- In general, the notebooks lack documentation of what exactly is done in it. This needs to be added for the notebooks to be useful.
- In `00_Grid/Grid.ipynb` : 
   ```
  ---------------------------------------------------------------------------
  ValueError                                Traceback (most recent call last)
  Input In [4], in <cell line: 15>()
       27 grid.initialize()
       29 #Kohn Sham object
  ---> 30 KS = Kohnsham(grid, Za, Zb, pol, Nmo, N, optKS)
       31 KS.scf(optKS)
       33 energy.append(KS.E.E)
  
  File ~/miniconda3/envs/t2a/lib/python3.8/site-packages/CADMium/kohnsham/kohnsham.py:54, in Kohnsham.__init__(self, grid, Za, Zb, pol, Nmo, N, optKS)
       52 for i in optKS.keys():
       53     if i not in KohnShamOptions().dict().keys():
  ---> 54         raise ValueError(f"{i} is not a valid option for KohnSham")
       55 optKS = KohnShamOptions(**optKS)
       57 #Options
  
  ValueError: verbose is not a valid option for KohnSham
   ```
- In `03_Inversion/Orbital_Bebe.ipynb`:
   ```
  An exception has occurred, use %tb to see the full traceback.
  
  SystemExit: Too many inversion failures. Stopping
  /home/sarath/miniconda3/envs/t2a/lib/python3.8/site-packages/IPython/core/interactiveshell.py:3406: UserWarning: To exit: use 'exit', 'quit', or Ctrl-D.
    warn("To exit: use 'exit', 'quit', or Ctrl-D.", stacklevel=1)
   ```

**Comment by VHchavez** [comment]
Hello Sarath, 

I have added a bit of information for each set of examples. They should provide some guidelines for inexperienced users. In general, I added each of the examples to the documentation. This redundancy should provide the same information no matter what source of information the user is using. 

I fixed the dependencies, making only matplotlib the only library used for plotting. This statement is now clearly specified in the README; thus, users should be aware of its need. 

Lastly, I fixed the notebooks with errors or other unnecessary cells.
